site_root/input/resources/terminology/CodeSystem-doid.json

```diff
@@ -0,0 +1,63 @@
+{
+    "resourceType": "CodeSystem",
+    "id": "doid",
+    "url": "http://purl.obolibrary.org/obo/doid.owl",
+    "name": "doid",
+    "title": "DOID",
+    "status": "draft",
+    "experimental": false,
+    "description": "DOID",
+    "caseSensitive": true,
+    "content": "complete",
```

Comment:
Please refer to https://www.hl7.org/fhir/valueset-codesystem-content-mode.html.

Reply:
Thanks.  Looking at the possibilities, I'd assume `fragment` is the correct value to use in this case?